src/blog/2024-10-14/index.md

```diff
@@ -0,0 +1,45 @@
+title: Neotokyo;Rebuild Development Progress Update
+summary: Update on project progress over the last couple of months
+author: kinoko and blaberry 
+
+# Introduction
+2024-10-14 kinoko and blaberry
+
+Hi, it has been quite a few months since the last dev progress update. Better late than never though, so let’s go over what has happened and where the project is heading!
+
+# What’s the state of NT;RE?
```

Comment:
Can you put `##`/2nd-level headers for all headers and add `#`/1st-level header on top as the title.

Reply:
For sure, ty!

Reply:
Done